Fix/admin menu

LatyshevDD opened the pull request:
- исправлено отображение кнопки "написать администратору" в компоненте DropDownMenu
- исправлено отображение элемента title__background  у кнопки выхода в компоненте DropDownMenu

Commit:
fix:
- исправлено отображение кнопки "написать администратору" в компоненте DropDownMenu
- исправлено отображение элемента title__background  у кнопки выхода в компоненте DropDownMenu

Commit:
Merge branch 'fix/admin-menu' of github.com:ya-pomogau/frontend into fix/admin-menu

marussia1996 reviewed (changes requested):
Логика отображения для разных ролей работает корректно, молодец:) Однако есть недочеты по верстке, посмотри пожалуйста повнимательнее на расположение меню на макете и у нас в проекте, оно различается 
![image](https://github.com/ya-pomogau/frontend/assets/29862392/40594e13-db8a-454c-a4ca-276efb152646)
![image](https://github.com/ya-pomogau/frontend/assets/29862392/e3009a8e-cff6-4915-89f3-e7b9664d2163)
Также обрати внимание, что при смене разрешения менюшка уплывает куда-то не туда) Надо подумать над тем как это решить, будут предложения - пиши, обсудим.
![image](https://github.com/ya-pomogau/frontend/assets/29862392/e80ef690-d190-424e-bdfc-5868397b816f)

Commit:
fix:
- исправлены расположение и размеры выпадающего меню
- исправлены размеры элементов выпадающего меню
- исправлено отображение разделительной линии между элементами выпадающего меню

Commit:
fix:
- для плавного изменения расположения выпадающего окна значени right установлено в относительных величинах

LatyshevDD commented:
Все поправил. Насчет плавности пока предлагаю просто установить значения отступов в относительной величине (реализовал в коде).

INextYP commented:
@LatyshevDD Привет! Касательно логики  работы - всё отлично!

@marussia1996 желательно, если есть косяки, которые не относятся к задаче (в рамках этой задаче нужно было убрать кнопку у админа и всё) заводить в беклог задачку с указанием проблемы. Давайте идти модульными шагами.

На текущий момент проблема с позиционированием сохраняется, так абсолютные величины жестко завязаны в коде.
@LatyshevDD если есть желание, то давай я создам тебе отдельную задачку (а этот пулл-реквест вмержим) в которой опишу, как лучше сделать позиционирование меню? :) Или, так как ты владеешь бекендом, возьми себе задачку на бекенд

LatyshevDD commented:
Да мне все равно что делать) Готов добить позиционирование в рамках отдельной задачи.

INextYP commented:
@LatyshevDD Можешь мержить) 

По следующей задаче, возьми задачу с бекенда, а то несбалансирован ход разработки проекта)
